src/NuGet.Core/NuGet.Protocol/Model/PackageSearchMetadataV2Feed.cs

```diff
@@ -138,6 +138,8 @@ private static Uri GetUriSafe(string url)
             return uri;
         }
 
+        public Task<PackageDeprecationMetadata> GetDeprecationMetadataAsync() => Task.FromResult<PackageDeprecationMetadata>(null);
```

Comment:
You can write a test for this.

Reply:
Added to the existing `PackageSearchMetadataV2Feed` tests.